apps/system/utils/utils_stackmonitor.c

```diff
@@ -109,7 +109,6 @@
  * Private Data
  ****************************************************************************/
 static volatile bool stkmon_started;
-
```

Comment:
You'd better keep previous blank line here.

Reply:
Okay. Thanks.